packages/logger-bunyan/lib/stream/BunyanStreamProviderFile.ts

```diff
@@ -0,0 +1,23 @@
+import {Stream} from "bunyan";
+import {BunyanStreamProvider, IBunyanStreamProviderArgs} from "./BunyanStreamProvider";
+
+/**
+ * A file bunyan stream provider.
+ */
+export class BunyanStreamProviderFile extends BunyanStreamProvider {
+
+  public readonly path: string;
+
+  constructor(args: IBunyanStreamProviderFileArgs) {
+    super(args);
+  }
+
+  public createStream(): Stream {
+    return { type: 'file', name: this.name, path: this.path.substr(8), level: this.level };
```

Comment:
I'm assuming this is to remove the `file:///` part. Might be a problem for Windows (not sure yet since I don't see this used anywhere?). I would suggest keeping the `file:///` part and converting it to a URL. `fs` accepts URLs (with the file protocol) for all its functions.

Reply:
👍